metafix/src/main/java/org/metafacture/metafix/Value.java

```diff
@@ -695,26 +698,26 @@ public String asString() {
             return map.toString();
         }
 
-        private enum InsertMode {
+    }
 
-            REPLACE {
-                @Override
-                void apply(final Hash hash, final String field, final String value) {
-                    hash.put(field, new Value(value));
-                }
-            },
-            APPEND {
-                @Override
-                void apply(final Hash hash, final String field, final String value) {
-                    final Value oldValue = hash.get(field);
-                    final Value newValue = new Value(value);
-                    hash.put(field, oldValue == null ? newValue : oldValue.merge(newValue));
-                }
-            };
+    private enum InsertMode {
 
-            abstract void apply(Hash hash, String field, String value);
+        REPLACE {
+            @Override
+            void apply(final Hash h, final String field, final String value) {
```

Comment:
Minor: Not particular fond of shortening the variable name.

Reply:
Avoiding the need to rename `hash` in https://github.com/metafacture/metafacture-fix/pull/78/commits/1904e0d5900f7410015a4a875a38cdb50a2b80db.